docs/examples.rst

```diff
@@ -0,0 +1,38 @@
+.. _aiohttp-examples:
+
+================
+Example Projects
+================
+
+Here are several fun projects, that can be used as template or
+inspiration of what can  be done with aiohttp. Please feel free to add your
+open source example project by making Pull Request.
+
+
+Official Example Projects
+-------------------------
+
+This list contains libraries which are supported by *aio-libs* team
+and located at https://github.com/aio-libs/aiohttp-demos
+
+- `shortify <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/shortify>`_
+  URL shortener with Redis storage.
+
+- `moderator <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/moderator>`_
+  UI and API for classification of offensive and toxic comments using Kaggle
+  data and scikit-learn.
+
+- `motortwit <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/motortwit>`_
+  Twitter clone with MongoDB storage.
+
+- `imagetagger <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/imagetagger>`_
+  Example how to deploy deep learning model with aiohttp.
+
+- `chat <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/chat>`_
+  Simple chat using websockets.
+
+- `polls <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/chat>`_
```

Comment:
replace link to `https://github.com/aio-libs/aiohttp-demos/tree/master/demos/polls`